### ISSUE TITLE
Add ARIA role=img and aria-label to alert/banner icons

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/component-library#191]
SODA reports that the alert/banner icons are non-decorative, and should therefore have meaningful labels.

For the icons in both [Alerts](https://sul-dlss.github.io/component-library/alerts/) and [Banner](https://sul-dlss.github.io/component-library/banners/), do one of the following:

- _Easy approach_: Add `role="img"` and `aria-label="Alert icon"` to each icon `<div>`.
- _Ideal approach_: Add `role="img"` and `aria-label="[alert type] icon"` to each icon `<div>`, corresponding to the alert type: Danger, Warning, Note, Info, Success.

<img width="400" height="32" alt="Image" src="https://github.com/user-attachments/assets/08bde853-4200-4c5a-a36c-d5280aa29a08" />

### Reported by SODA
> [Summary]
> Text alternative for the informative image is missing
> 
> [Description]
> The text alternative for the informative image is missing.
> There is an alert icon that is informative. As it is posted as a CSS background image, there is no text alternative to this icon. 
> 
> [Impact]
> Critical
> 
> [Recommended to fix]
> Add a text equivalent using one of many different techniques. One would be to add an aria-label to the div as well as a role to make the DIV have semantic meaning. 
> 
> `<div class="fs-3 me-3 align-self-center d-flex justify-content-center bi bi-exclamation-triangle-fill" role="img" aria-label="Alert">`
> 
> Rule : 
> Images that convey content MUST have programmatically-discernible alternative text.
> 
> How To Fix: 
> Fix this issue by using ONE of the following techniques:
> 
> 1. Use aria-label and role="img" to convey the intent, purpose, or meaning of the image.
> 
> `<p>Have a great day <span class="fonticon smiley" role="img" aria-label="Smiley face"></span></p>`
> 
> `<svg role="img" aria-label="cat">`
> 
> 2. Use CSS clipped text to provide screen reader accessible (non-visible) alternative text.
> 
> ```
> <a href="https://www.facebook.com/dequesystems" class="fb">
> <span class="sr-text">Deque's Facebook page</span>
> </a>
> 
> .sr-text {
> position: absolute;
> clip: rect(1px, 1px, 1px, 1px);
> -webkit-clip-path: inset(50%) ;
> clip-path: inset(50%);
> padding: 0;
> border: 0;
> height: 1px;
> width: 1px;
> white-space: nowrap;
> overflow: hidden;
> }
> ```
> 
> Reference: 
> Deque University: https://dequeuniversity.com/class/images/alt-text/informative
> W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/
> 
> Background: 
> People who are blind cannot see images on a page. In order to give people who cannot see an image access to the information conveyed by the image, it must have a text alternative. The text alternative must describe the information or function represented by the image. Screen readers can then use the alternative text to convey that information to the screen reader user.
> 
> For informative images - images that convey information - the alternative text must communicate the intent, purpose, or meaning of the image in a way that serves as a true alternative for the image.
> 
> It is not possible to provide alternative text by adding an alt attribute directly to CSS background images. Images that convey content should not generally be placed in CSS. However, if it is necessary to do so, you can add alternative text to informative and actionable CSS images using other methods.
> 
> [Source Code]
> element 1 : 
> ```
> <div class="alert d-flex shadow-sm align-items-center alert-danger">
>     <div class="fs-3 me-3 align-self-center d-flex justify-content-center bi bi-exclamation-triangle-fill"></div>
>   <div class="text-body">
>     You must complete sections 1-7 before you can review and submit to the Registrar.
>   </div>
> </div>
> ```
>
